src/core/dev/rfs_uc.cpp

```diff
@@ -129,6 +129,27 @@ bool rfs_uc::prepare_flow_spec()
         }
     }
 #endif
+#if defined(DEFINED_ENVOY)
+    else if (g_p_app->workers_num > 0 && g_p_app->get_worker_id() >= 0) {
+        if (m_flow_tuple.get_protocol() != PROTO_UDP) {
+            int src_port;
+            if (g_p_app->add_second_4t_rule) {
```

Comment:
what is the purpose of second 4t rule?

Reply:
Add 4 tuple rules that are using TCP.sport in order to distribute the TCP connection between multiple workers. The add_second_4t_rule is needed for case when number of workers not power of two